extensions/wikia/AdEngine/js/spec/provider/gpt/adDetect.spec.js

```diff
@@ -21,12 +25,19 @@ describe('Method ext.wikia.adEngine.provider.gpt.adDetect.onAdLoad', function ()
 				}
 			};
 
-			gptHop = modules['ext.wikia.adEngine.provider.gpt.adDetect'](mocks.log, mocks.window, mocks.adContext);
+			mocks.iframe.contentWindow = {
+				'AdEngine_adType': forceAdType,
```

Comment:
It has to equal specific value or undefined, due to (this if statement)[https://github.com/Wikia/app/blob/aden-2052-collapse-adtype/extensions/wikia/AdEngine/js/provider/gpt/adDetect.js#L106] in getAdType method.